src/api/customer/customer-business.ts

```diff
@@ -11,6 +11,34 @@ const persistCustomer: CustomerCreationRequestHandler = async (req, res, next) =
     }
 }
 
+const persistFindAllCustomers: CustomersFindAllRequestHandler = async (req, res, next) => {
+    
+    try {       
+        const off = req.query.offset
+        const lim = req.query.limit
+        const users:Customer[] = await Customer.findAll({limit: parseInt(lim) || undefined, offset: parseInt(off) || undefined});
+        res.locals.customerToFindAll = users;
+        next();
+    } catch(error) {
+        next(error);
+    }
+}
+
+const persistFindCustomers: CustomersFindRequestHandler = async (req, res, next) => {
+    try {
+        const { customerToFind } = res.locals;
+        const users = await Customer.findByPk(customerToFind.uuid);
+        if(users != null){
```

Comment:
Mesma coisa, simplificação de um `falsy value`.
```suggestion
        if (users) {
```